modules/storage-persistent-storage-overview.adoc

```diff
@@ -8,38 +8,19 @@
 [id=persistent-storage-overview_{context}]
 = Persistent storage overview
 
-Managing storage is a distinct problem from managing compute resources.
-{product-title} uses the Kubernetes persistent volume (PV) framework to
-allow cluster administrators to provision persistent storage for a cluster.
-Developers can use persistent volume claims (PVCs) to request PV resources
+Managing storage is a distinct problem from managing compute resources. {product-title} uses the Kubernetes persistent volume (PV) framework to allow cluster administrators to provision persistent storage for a cluster. Developers can use persistent volume claims (PVCs) to request PV resources
 without having specific knowledge of the underlying storage infrastructure.
 
-PVCs are specific to a project, and are created and used by developers as
-a means to use a PV. PV resources on their own are not scoped to any
-single project; they can be shared across the entire {product-title}
-cluster and claimed from any project. After a PV is bound to a PVC,
-that PV can not then be bound to additional PVCs. This has the effect of
-scoping a bound PV to a single namespace, that of the binding project.
+PVCs are specific to a project, and are created and used by developers as a means to use a PV. PV resources on their own are not scoped to any single project; they can be shared across the entire {product-title} cluster and claimed from any project. After a PV is bound to a PVC, that PV can not then be bound to additional PVCs. This has the effect of scoping a bound PV to a single namespace, that of the binding project.
 
-PVs are defined by a `PersistentVolume` API object, which represents a
-piece of existing storage in the cluster that was either statically provisioned
-by the cluster administrator or dynamically provisioned using a `StorageClass` object. It is a resource in the cluster just like a
-node is a cluster resource.
+PVs are defined by a `PersistentVolume` API object, which represents a piece of existing storage in the cluster that was either statically provisioned by the cluster administrator or dynamically provisioned using a `StorageClass` object. It is a resource in the cluster just like a node is a cluster resource.
 
-PVs are volume plugins like `Volumes` but
-have a lifecycle that is independent of any individual pod that uses the
-PV. PV objects capture the details of the implementation of the storage,
-be that NFS, iSCSI, or a cloud-provider-specific storage system.
+PVs are volume plugins like `Volumes` but have a lifecycle that is independent of any individual pod that uses the PV. PV objects capture the details of the implementation of the storage, be that NFS, iSCSI, or a cloud-provider-specific storage system.
```

Comment:
```suggestion
PVs are volume plugins like `Volumes` but have a lifecycle that is independent of any individual pod that uses the PV. PV objects capture the details of the implementation of the storage, be that LVM, the host filesystem (i.e. hostpath) or raw block devices.
```

I figure it's better to use storage that doesn't inherently support network access as examples.